integrations/quickbooks/nango.yaml

```
@@ -320,3 +332,19 @@ models:
         customer_ref: Reference
         currency_ref?: Reference
         project_ref?: Reference
+    GeneralLedger:
+        id: string
+        date: string
+        createdDate: string
+        updatedDate: string
+        currency: string
+        note: string
+        lines: LedgerLine[]
+    LedgerLine:
+        journalLineId: string
+        type: string
+        accountId: string
+        accountName: string
+        netAmount: number
+        postingType: 'Debit | Credit'
```

If this is meant to be an enum it needs to be expressed like this:
```suggestion
        postingType: Debit | Credit
```

---

noted and updated